.github/ISSUE_TEMPLATE/01_bug_report.md

```diff
@@ -0,0 +1,30 @@
+# 🐛 Bug report
+
+---
+name: 🐛 Bug report
+about: If something isn't working 🤕
+
+---
+
+**Context**
```

Comment:
What shall be added to Context? Maybe the nock version used?

Reply:
The bug report has a versions heading where you can list your versions of Node and Nock. I agree that this can be to much. I'll remove this one. :)